lib/metadata/util/md5deep.rb

```diff
@@ -139,12 +139,15 @@ def processFile(path, x, xmlNode)
 
           ext = File.extname(currFile).downcase
           if @opts.winVerList.include?(ext)
-            if @opts.versioninfo || @opts.imports
+            begin
               peHdr = PEheader.new(fh) rescue nil
               unless peHdr.nil?
                 xmlFileNode.add_element("versioninfo", peHdr.versioninfo) if @opts.versioninfo && !peHdr.versioninfo.blank?
                 xmlFileNode.add_element("libraries", "imports" => peHdr.getImportList) if @opts.imports && !peHdr.imports.blank?
               end
+            rescue TypeError => err
+              $log.debug "processFile: TypeError handling PEheader; skipping PEheader info"
```

Comment:
Should this be debug or info? Will we want to see that this happened in a non-debug log?
Maybe just the backtrace part should be debug.

Reply:
@roliveri I can do that.  A lot easier than dealing with codeclimate nesting complaints.